src/test/scala/edu/ie3/simona/test/common/input/StorageInputTestData.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * © 2022. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.simona.test.common.input
+
+import edu.ie3.datamodel.models.OperationTime
+import edu.ie3.datamodel.models.input.OperatorInput
+import edu.ie3.datamodel.models.input.system.StorageInput
+import edu.ie3.datamodel.models.input.system.`type`.StorageTypeInput
+import edu.ie3.datamodel.models.input.system.characteristic.CosPhiFixed
+import edu.ie3.simona.test.common.DefaultTestData
+import edu.ie3.util.quantities.PowerSystemUnits._
+import tech.units.indriya.quantity.Quantities
+
+import java.util.UUID
+
+trait StorageInputTestData extends DefaultTestData with NodeInputTestData {
+
+  protected val storageTypeInput = new StorageTypeInput(
+    UUID.fromString("fbee4995-24dd-45e4-9c85-7d986fe99ff3"),
+    "Dummy_StorageTypeInput",
+    Quantities.getQuantity(100d, EURO),
+    Quantities.getQuantity(101d, EURO_PER_MEGAWATTHOUR),
+    Quantities.getQuantity(200d, KILOWATTHOUR),
+    Quantities.getQuantity(13d, KILOVOLTAMPERE),
+    0.997,
+    Quantities.getQuantity(12.961, KILOWATT),
+    Quantities.getQuantity(0.03, PU_PER_HOUR),
+    Quantities.getQuantity(0.92, PU),
+  )
```

Comment:
```suggestion
  protected val storageTypeInput = new StorageTypeInput(
    UUID.fromString("fbee4995-24dd-45e4-9c85-7d986fe99ff3"),
    "Dummy_StorageTypeInput",
    Quantities.getQuantity(15000d, EURO),
    Quantities.getQuantity(0.05d, EURO_PER_MEGAWATTHOUR),
    Quantities.getQuantity(20d, KILOWATTHOUR),
    Quantities.getQuantity(10d, KILOVOLTAMPERE),
    0.997,
    Quantities.getQuantity(9.97, KILOWATT),
    Quantities.getQuantity(0.03, PU_PER_HOUR),
    Quantities.getQuantity(0.92, PU),
  )
```
This is very optional, but I would like to provide realistic test data for two reasons: 

1. users / students get a feeling for realistic values
2. we may find bugs faster since values seems to be unrealistic

Reply:
I agree, we'll just have to adapt a bunch of tests then.

Reply:
True, I'm fine with the 200 kWh Storage. Please just adapt the costs :)

Reply:
Done